src/test/java/racingcar/CarTest.java

```diff
@@ -0,0 +1,34 @@
+package racingcar;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+class CarTest {
+    @DisplayName("자동차의 이름은 5글자를 초과하면 예외가 발생한다")
+    @Test
+    void constructor() {
+        assertThatIllegalArgumentException()
+            .isThrownBy(() -> new Car("동해물과백두산이"));
```

Comment:
경계값을 테스트해보는 것이 좋다고 생각해요!
예를들면 아래와 같이 초과하는 가장 근접한 값인 6자리로 테스트해보는 것이죠

```suggestion
            .isThrownBy(() -> new Car("여섯글자이름"));
```